File: Day-06/RunningSum.py
nums = [3, 1, 2, 10, 1];


def runningSum(nums):
    length = len(nums)
    newArray = [nums[0]]
    for i in range(1, length):
        newArray.append( newArray[i-1] + nums[i])
        print(i)
    
    print(newArray)


runningSum(nums)

File: Day-06/RunningSum.js
const nums = [3, 1, 2, 10, 1];

function runningSum(inputArray) {
  const lengthOfArray = inputArray.length;
  const newArray = [];
  newArray[0] = inputArray[0];
  for (let i = 1; i < lengthOfArray; i++) {
    newArray[i] = newArray[i - 1] + inputArray[i];
  }

  return newArray;
}

console.log(runningSum(nums));
